lms/services/lti_outcomes.py

```diff
@@ -118,11 +50,19 @@ def read_result(self, outcomes_request_params):  # pylint:disable=no-self-use
 
         :return: The last-submitted score or `None` if no score has been submitted.
         """
-        body = READ_RESULT_REQUEST_TEMPLATE.render(
-            lis_result_sourcedid=outcomes_request_params.lis_result_sourcedid
-        )
 
-        result = self._send_request(outcomes_request_params, body)
+        result = self._send_request(
+            outcomes_request_params,
+            request_body={
+                "readResultRequest": {
+                    "resultRecord": {
+                        "sourcedGUID": {
+                            "sourcedId": outcomes_request_params.lis_result_sourcedid
+                        }
+                    }
+                }
+            },
```

Comment:
A lot cleaner than the previous `READ_RESULT_REQUEST_TEMPLATE`, and also inlined in the place where it's used :+1: